### ISSUE TITLE
Support for hapi 13.0.0

### ISSUE THREAD

**Issue by robert-mendoza** [mac-/hapi-logger#5]
Very nice logger you have here.  Any chance it will be updated to the current version of hapi?

Thanks!

**Comment by mac-** [comment]
yeah, sorry for the slow response. I can look into updating this in the next couple days.

**Comment by robert-mendoza** [comment]
That would be great... I have been using hapi-logger with Hapi 13.0.0 without an issue, so should be just a case of updated the dependencies. Thanks!
